monai/transforms/spatial/dictionary.py

```diff
@@ -404,15 +420,26 @@ def __init__(
         """
         Args:
             keys: keys of the corresponding items to be transformed.
-            spatial_size (2 ints): specifying output image spatial size [h, w].
+            spacing: distance in between the control points.
+            magnitude_range: 2 int numbers, the random offsets will be generated from
+                ``uniform[magnitude[0], magnitude[1])``.
+            spatial_size: specifying output image spatial size [h, w].
```

Comment:
If the components of the `spatial_size` are non-positive values, the transform will use the corresponding components of `img` size. For example, `spatial_size=(32, -1)` will be adapted to `(32, 64)` if the second spatial dimension size of `img` is `64`.